src/main/java/htsjdk/samtools/cram/CRAIEntry.java

```diff
@@ -75,28 +84,33 @@ public void writeToStream(OutputStream os) {
     private String serializeToString() {
         return String.format(entryFormat,
                 sequenceId, alignmentStart, alignmentSpan,
-                containerStartOffset, sliceOffset, sliceSize);
+                containerStartOffset, sliceByteOffset, sliceByteSize);
+    }
+
+    /**
+     * Create a SliceBAIMetadata object from a CRAIEntry by supplying the necessary fields
```

Comment:
BAI stores information that CRAI does not, so we need to add the two missing fields here